### ISSUE TITLE
Pandas FutureWarning

### ISSUE THREAD

**Issue by alexandre-dang** [tensorflow/decision-forests#135]
`FutureWarning: In a future version of pandas all arguments of DataFrame.drop except for the argument 'labels' will be keyword-only.`
in tensorflow_decision_forests/keras/core_inference.py:873 and probably at other places
`"df = df.drop(label, 1)"` should be replaced by `"df = df.drop(label, axis=1)"`

**Comment by rstz** [comment]
Hi,

thank you for the report, a fix is on its way!
